.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+### Description of Changes
+
+(briefly outline the reason for changes, and describe what's been done)
+
+### Breaking Changes
+
+- None
+
+### Release Checklist
+
+Prepare:
+
+- [ ] Detail any breaking changes. Breaking changes require a new major version number, and a migration guide in wiki / README.md
+- [ ] Check `./gradlew assemble` passes w/ no errors
+
+Bump versions in:
+
+- [ ] CHANGELOG.md
+
+Integration tests:
+
+## T&T Only
+
+- [ ] Init SDK with only optimove tenant / config
+- [ ] Track events
+
+## Push Only
+
+- [ ] Init SDK with all credentials
+- [ ] Track events
+- [ ] Associate customer (verify both backends)
+- [ ] Register for push
+- [ ] Opt-in for In-App
+- [ ] Send test push
+- [ ] Send test In-App
+- [ ] Receive / trigger deep link handler (In-App/Push)
+- [ ] Receive / trigger the content extension, render image and action buttons for push
+- [ ] Verify push opened handler
+
+## Deferred Deep Links
+
+- [ ] With app installed, trigger deep link handler
+- [ ] With app uninstalled, follow deep link, install test bundle, verify deep link read from Clipboard, trigger deep link handler
+
+## Combined
+
+- [ ] Track event for T&T, verify push received
+- [ ] Trigger scheduled campaign, verify push received
+- [ ] Trigger scheduled campaign, verify In-App received
+
+Release:
```

Comment:
Worth looking at heading levels on tests vs the steps? Perhaps we could just use bold for cases or something?

Are these steps complete -- not sure about how this SDK is published yet. (any tags to make etc.)

Reply:
I assume they are not, @KonstantinOptimove can probably document / complete this step.

Reply:
For the headings i've done a pass to smooth out the heading levels.